src/main/diagnostics/abstractCodeActionProvider.ts

```diff
@@ -89,14 +89,34 @@ export abstract class AbstractCodeActionProvider implements vscode.CodeActionPro
             let diagnostic: vscode.Diagnostic = new vscode.Diagnostic(
                 new vscode.Range(dependencyPos[i], dependencyPos[i + 1]),
                 node.topSeverity === Severity.Normal ? 'No issues found.' : 'Top issue severity: ' + SeverityUtils.getString(node.topSeverity),
-                DiagnosticsUtils.getDiagnosticSeverity(node.topSeverity)
+                vscode.DiagnosticSeverity.Hint
```

Comment:
Why did you delete this? I liked this feature.

Now the problems view is empty, but before this change it was:
![image](https://user-images.githubusercontent.com/11367982/157414782-dc152a9d-035a-4db1-84c4-772493a8096e.png)